loadtest/user/userentity/actions.go

```diff
@@ -335,7 +335,7 @@ func (ue *UserEntity) SearchChannels(teamId string, search *model.ChannelSearch)
 }
 
 func (ue *UserEntity) GetChannelsForTeamForUser(teamId, userId string) ([]*model.Channel, error) {
-	channels, resp := ue.client.GetChannelsForTeamForUser(teamId, userId, "")
+	channels, resp := ue.client.GetChannelsForTeamForUser(teamId, userId, false, "")
```

Comment:
I think we need this `bool` to be a parameter of the function. The webapp client currently calls this with `true`.

At least here we can break the API for now :stuck_out_tongue: